lib/helpers.js

```diff
@@ -309,7 +309,10 @@ helpers.attemptTraversal = function (node, traversalPath) {
       currentNodeList = [],
       processChildNode = function processChildNode (child) {
         child.forEach(traversalPath[i], function (n) {
-          nextNodeList.push(n);
+          if (n.content && n.content.some && n.contains('interpolation')) {
```

Comment:
Are you using `some` here to determine if the array has length? Normally `Array.prototype.some` requires a callback function - if above suggest just using `n.content.length`

Reply:
Well content might be a string and a string still has a length property so that's not a good way to determine if it's an array. We're not invoking `some` just checking if it's defined on the prototype of whatever is in content and stops the rule crashing if it then tries to use the contain method which doesn't exist on a content of type string.

Reply:
Why not just use isArray?

Reply:
Yeah i'll be honest i dont really know why i did it this way. I think I was actually doing something else with a `some` at the same time and decided to use it. odd.. will change it now anyways.